### PR TITLE
[AR] Translation of 14-2.md 

### PR DESCRIPTION
**Pull request by anasselhoud**
There was an error with merging the last PR. So I closed it and created a new one.

### CODE REVIEW

**Comment by hayaalsh** [comment]
Great job Anass! Honestly there is nothing much to change. Only few comments on the terms.

- Could you please add your translation of the following to the terms table: Sliding window, sigmoid, hinge, forward algorithm, belief-propagation algorithm, log sum exponential, negative log-likelihood, negative log-likelihood loss, log sum exponential, Lagrangian, entropy.
- Terms that do not translate, e.g., softmax, Viterbi, can be add to the table. But set their translation to their same English name (see term "Adaline").
- Line 93:  remove ة to get "أكبر من بعض القيم" instead of "أكبر من بعض القيمة"
- Line 171: add your translation of "Negative Log-Likelihood" along with the English name in brackets as your did in line 264.
- Line 410: "خاصية لمجموعة التوزيع" instead of "خاصية لعائلة التوزيع" 
- Line 412: "معاملات النموذج" instead of "معلمات النموذج"
- Line 416: "حساب سجل مجموع الأسي log of a sum of exponentials" add brackets, or remove the English term. You can add the English term on line 200 where the term was first mentioned in your text.

File: docs/ar/week14/14-2.md
Comment (hayaalsh) [inline docs/ar/week14/14-2.md:27]: Add Arabic translation of Figure 1 caption.

File: docs/ar/week14/14-2.md
Comment (hayaalsh) [inline docs/ar/week14/14-2.md:354]: Translate Figure 6 caption.

**Review by hayaalsh — CHANGES_REQUESTED** [review]
Great job Anass! Honestly there is nothing much to change. Only few comments on the terms and some missing captions.

- Could you also add your translation of the following to the terms table: sliding window, sigmoid, hinge, forward algorithm, belief-propagation algorithm, log sum exponential, negative log-likelihood, negative log-likelihood loss, Lagrangian, entropy.
- Terms that do not translate, e.g., softmax, Viterbi, can be add to the table. But set their translation to their same English name (see term "Adaline").

**Comment by anasselhoud** [comment]
@hayaalsh  That's not how we work here with Github. Reviewing is a step apart. When @Atcold merge the PR, you can send a new pull request with all the changes.

**Comment by anasselhoud** [comment]
Thank you @hayaalsh for your comments. Please wait until @Atcold merge the PR, you can send a new pull request with all the changes so it counts as your proper contribution too (if I changed now, it will be counted as my contribution alone.

**Review by Atcold — APPROVED** [review]
Great! Please send a new PR with the revised content.

**Pull request merged**
